src/Vlingo.Common/Vlingo.Common.csproj

```diff
@@ -34,7 +35,18 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.3.1" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.0.0" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
-    <PackageReference Include="Vlingo.UUID" Version="0.6.4.6" />
   </ItemGroup>
+  <Choose>
+    <When Condition="'$(Configuration)' == 'Debug With Project References'">
+      <ItemGroup>
+        <ProjectReference Include="..\..\..\vlingo-net-uuid\src\Vlingo.UUID\Vlingo.UUID.csproj" />
+      </ItemGroup>
+    </When>
+    <Otherwise>
+      <ItemGroup>
+        <PackageReference Include="Vlingo.UUID" Version="$(VlingoVersion)" />
```

Comment:
This is not exactly that as we can have different versions for each vlingo. But I see the patter. Would be easy to add in `Directory.Build.props`